src-tauri/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "refstudio"
 version = "0.0.0"
-description = "A Tauri App"
+description = "An open source text editor optimized for writing that relies on references."
 authors = ["you"]
-license = ""
-repository = ""
+license = "MIT"
+repository = "https://github.com/refstudio/refstudio"
 edition = "2021"
```

Comment:
Shouldn't this be 2023?

Reply:
No. that's a cargo configura! Go figure why the have their fields alongside with fields we can configure.